src/helpers/verification.py

```diff
@@ -74,6 +74,28 @@ async def _check_for_ban(uid: str) -> Optional[Dict]:
 
     return ban_details
 
+async def process_certification(certid: str, name: str):
+    cert_api_url = f"{settings.API_V4_URL}certificate/lookup"
```

Comment:
There is a `/` missing

```suggestion
    cert_api_url = f"{settings.API_V4_URL}/certificate/lookup"
```